src/main/webapp/components/search-forms/container.tsx

```diff
@@ -99,13 +117,20 @@ const useCreate = () => {
         .filter(({ id }: { id: string }) => id !== data.createMetacard.id)
         .concat(data.createMetacard)
 
+      const user = getIn(cache.readQuery({ query }), ['user'], {})
+      const { email, roles } = user
```

Comment:
Does this break default search forms? Not sure if preferences needs to be updated

Reply:
So I didn't include user prefs here, because there's no way to automatically update the cache when those change since there's no id associated with it. There was an error in the console on master that the `preferences` was missing for the cache update for this. I'm gonna look into a way to see if there's a better way to do it.

Reply:
I'm a little confused by this statement `there's no way to automatically update the cache when those change since there's no id associated with it`. Isn't this a reason why we would manually update it here?

Maybe I'm misunderstanding the problem, but it seems like you could pull preferences and add it to your update of the cache. Something like: 
 ```
user: {
    email,
    roles,
    preferences: {
        {...preferences}
        __typename: 'UserPreferences'
    }
}
```